devel/ci/bodhi-ci

```diff
@@ -294,6 +312,44 @@ def integration(archive, concurrency, container_runtime, no_build, failfast, ini
                                archive_path=archive_path))
 
 
+class ProgressReporter(object):
```

Comment:
I guess we can just rebase this PR when the other PR with this code merges.

Reply:
Agreed.